### ISSUE TITLE
Add steps at runtime

### ISSUE THREAD

**Issue by Ryggl** [elclanrs/jq-idealforms#51]
Hi!
Is there any way to add steps at runtime in a form depending on a number I put in an input field? I want to add components to an order depending on the number a user put in. I tried to clone the sections in a loop, but the steps are loaded at the beginning of loading the page and I don't know how to refresh it at runtime if I cloned the sections.

**Comment by kingofnull** [comment]
yes it's big problem and the code is too complex to add the function. To do it u should add a section and then call a function named '_buildNavItems' but its impossible to do cause it's a protected function. I hope main developer solve this or at list describe how add public functions.
